flang/test/Semantics/OpenMP/do08.f90

```diff
@@ -135,4 +137,33 @@ program omp
   end do foo
   !$omp end do
 
+  !$omp do collapse(3)
+loopk: do k=1,10
+  loopj: do j=1,10
+    loopi: do i=1,10
+      ifi : if (.true.) then
```

Comment:
Nit: the indentation in these lines should be increased by one level.

Reply:
Changed this as well as the indentation for the whole loop nest.